feat/Railways

Nakatai-0322 opened the pull request:
[FREESERVER Railways](https://deploy-preview-140--freeserver-wiki.netlify.app/transports/train/fr/)に関する記事と、民鉄の鉄道記事の参考として、[Example Railways](https://deploy-preview-140--freeserver-wiki.netlify.app/transports/train/example/)(ER)という架空の路線群、組織に関する記事を書きますた。

Nakatai-0322 committed:
Add FR

Nakatai-0322 committed:
Update index.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
もよりすぽっと。

Nakatai-0322 committed:
Merge branch 'master' into add-trainArticle-1

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Create guide-bell.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Delete guide-bell.md

Nakatai-0322 committed:
Create guide-bell.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Update index.md

Nakatai-0322 committed:
Create FRICA.md

Nakatai-0322 committed:
Update index.md

Nakatai-0322 committed:
Update index.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update EW.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Update N.md

Nakatai-0322 committed:
Update index.md

Nakatai-0322 committed:
Merge branch 'master' into add-trainArticle-1

Nakatai-0322 commented:
@Alpha9n 
VCでやるって言ってたんでれびゅわー指名しました。

Nakatai-0322 committed:
[path]FR=>fr

Nakatai-0322 committed:
file path rename

Nakatai-0322 committed:
Delete docs/transports/train/FR directory

Nakatai-0322 committed:
Create ew.md

Nakatai-0322 committed:
Create n.md

Nakatai-0322 committed:
path resolve

Nakatai-0322 committed:
path resolve(2)

Nakatai-0322 committed:
Delete N.md

Nakatai-0322 committed:
Delete EW.md

Nakatai-0322 committed:
Update frica.md

Nakatai-0322 committed:
Update frica.md

Nakatai-0322 committed:
Merge branch 'master' into add-trainArticle-1

Nakatai-0322 committed:
Add example railways.

Nakatai-0322 committed:
Update ws.md

Nakatai-0322 committed:
Update cl.md

Nakatai-0322 committed:
Update index.md

Nakatai-0322 committed:
Update ws.md

Nakatai-0322 committed:
Update ws.md

Nakatai-0322 committed:
Update index.md

Nakatai-0322 committed:
Update cl.md

Nakatai-0322 committed:
Update cl.md

Nakatai-0322 committed:
Update index.md

Nakatai-0322 committed:
Update ws.md

Nakatai-0322 committed:
Update cl.md

Nakatai-0322 committed:
Update index.md

Nakatai-0322 committed:
Update cl.md

graycat27 commented on docs/item/frica.md:
```diff
@@ -0,0 +1,40 @@
+
+## 外観
+
+![FRICA](https://cdn.jsdelivr.net/gh/freeserverproject/texture/pack/assets/minecraft/textures/item/frica2.png)
```
GitHubのテクスチャリポジトリにある以下の素材を利用してもいいのでは？
https://github.com/freeserverproject/texture/blob/master/pack/assets/minecraft/textures/item/frica2.png

Nakatai-0322 replied:
まったく同じ素材です。jsdelivrで拾ってきてるんで、拾い方の違いだけですね。

ですが、規定でImgurを使うことになっていたので、そこは直しておきます。

Nakatai-0322 replied:
ついでに上下の余白もチョキチョキしときまつ。

Nakatai-0322 replied:
かんりお。
https://github.com/freeserverproject/freeserver-wiki/commit/aa187a7c3885f7f1450eda0561ae27c3b5a4d519

graycat27 commented on docs/item/frica.md:
```diff
@@ -0,0 +1,40 @@
+
+[FR](/transports/train/fr/)全線全駅で購入できます。
+
+券売機で、「記名済みFricaを購入する」もしくは「Fricaを購入する」を押して記名済みでないものが出てきたら券売機で「記名する」を押して記名します。
```
```
券売機で「記名済みFricaを購入する」ボタンを押します。
もしくは、無記名のFricaを購入するか、他の人から受け取ってから、券売機の「記名する」ボタンで記名します。
Fricaの利用には記名が必要です。

Nakatai-0322 replied:
✔️: https://github.com/freeserverproject/freeserver-wiki/pull/140/commits/61f8ad682ba1a03cbdd77625fc9c1533b04ce9c8 , https://github.com/freeserverproject/freeserver-wiki/pull/140/commits/e617e8ef3874880f3fb140a613e2bd3e6843b87d

graycat27 commented on docs/item/frica.md:
```diff
@@ -0,0 +1,40 @@
+
+### 駅の場合
+
+改札機にかざして、ゲートが開いたら駅構内に行きましょう。
```
出るときもね

Nakatai-0322 replied:
✔️: https://github.com/freeserverproject/freeserver-wiki/commit/61f8ad682ba1a03cbdd77625fc9c1533b04ce9c8 , https://github.com/freeserverproject/freeserver-wiki/commit/e617e8ef3874880f3fb140a613e2bd3e6843b87d

graycat27 commented on docs/transports/train/example/cl.md:
```diff
@@ -0,0 +1,46 @@
+# 中心環状線
+
+[ER](/transports/train/example/)中心環状線（`central loop line`, イーアールちゅうしんかんじょうせん）とは、FREESERVER [メインワールド](/world/main/)のメインワールドを円状に結ぶ**架空の**鉄道路線です。
```
`
FREESERVER [メインワールド](/world/main/)のメインワールドを
`

Nakatai-0322 replied:
https://github.com/freeserverproject/freeserver-wiki/commit/9fe38b90c69cd81dca7bbe1756e7144e70d5ef32
にて修正。

眠たいときにWiki書くことは鯖民さんの混乱しかもたらさんな。

graycat27 commented on docs/transports/train/fr/ew.md:
```diff
@@ -0,0 +1,33 @@
+|周磋|Shūsa|`EW48`||Yes|
+|樫木町|KashigiTown|`EW49`||No|
+|中心町|ChūshinTown|`EW50`|メインワールド スポーン地点, FR南北線のりば（`N10`）|No|
+|中心町西|ChūshinTown-East|`EW51`||No|
```
![image](https://user-images.githubusercontent.com/64903817/212907247-e2c5ee60-8bcc-4b8a-9786-cbeeb31f57bf.png)
ローマ字表記（TownとかVil.Southが混ざってる…）よりも、鯖内で掲出されている正式英語表記の採録が良さそう

Nakatai-0322 replied:
https://github.com/freeserverproject/freeserver-wiki/pull/140/commits/9625dd4bcaa63dbd9b8696035f5ba15ee8bd7eb3

Nakatai-0322 replied:
南北線はまだできてないので今日あたり採録します。

Nakatai-0322 replied:
こんぷりーてぃっど。

https://github.com/freeserverproject/freeserver-wiki/pull/140/commits/5075bc8445c2186c46579067086ade5a1936c478

graycat27 commented on docs/transports/train/fr/ew.md:
```diff
@@ -0,0 +1,33 @@
+|灰猫村南|HainekoVil.-South|`EW47`||Yes|
+|周磋|Shūsa|`EW48`||Yes|
+|樫木町|KashigiTown|`EW49`||No|
+|中心町|ChūshinTown|`EW50`|メインワールド スポーン地点, FR南北線のりば（`N10`）|No|
```
`
南北線
`
リンクにしたい

Nakatai-0322 replied:
✔️: https://github.com/freeserverproject/freeserver-wiki/commit/c794219478465a8238cba1f2aa993be50c055427

graycat27 commented on docs/transports/train/fr/n.md:
```diff
@@ -0,0 +1,34 @@
+|駅名|駅名ローマ字表記|駅コード|最寄りスポット|
+|---|---|---|---|
+|牟志那駅|Mujina|`N09`|白玉屋|
+|中心町駅|ChūshinTown|`N10`|メインワールド スポーン地点, FR東西線のりば（`EW50`）|
```
`
東西線
`
リンクにしたい

Nakatai-0322 replied:
✔️: https://github.com/freeserverproject/freeserver-wiki/commit/f51b1cc03906eae6fc7e28905e9eaf48d4e2109e

graycat27 commented on docs/transports/train/guide-bell.md:
```diff
@@ -0,0 +1,18 @@
+# 盲導鈴
+
+盲導鈴とは、誘導が必要な方を鉄道駅などに誘導するための音声誘導装置である。
```
`
鉄道駅などで、視覚に拠らない誘導を必要とする人を出入口などに誘導する音声装置
`

Nakatai-0322 replied:
✔️: https://github.com/freeserverproject/freeserver-wiki/commit/a4251b219a3555df0526cd2d733d31734d4586ab

graycat27 commented on docs/transports/train/guide-bell.md:
```diff
@@ -0,0 +1,18 @@
+<!--
+## 仕様公開
+
+仕様公開の許可取り＆回路のSS撮って追記する。
```
@A-B-S-I 許可求められてますよ

graycat27 replied:
@A-B-S-I remind

Nakatai-0322 committed:
Update frica.md

https://github.com/freeserverproject/freeserver-wiki/pull/140#discussion_r1072179700
の意見にある説明不足の解決

Nakatai-0322 committed:
Update cl.md

Nakatai-0322 committed:
Update ew.md

Nakatai-0322 committed:
Update n.md

Nakatai-0322 committed:
Update ew.md

Nakatai-0322 committed:
Update guide-bell.md

Nakatai-0322 committed:
Update ew.md

Nakatai-0322 committed:
Update frica.md

Nakatai-0322 committed:
Update index.md

Nakatai-0322 committed:
Update cl.md

Nakatai-0322 committed:
Update cl.md

Nakatai-0322 committed:
Update cl.md

Nakatai-0322 committed:
Update frica.md

Nakatai-0322 committed:
Update frica.md

Nakatai-0322 committed:
Update n.md

graycat27 commented on docs/item/frica.md:
```diff
@@ -0,0 +1,77 @@
+
+![出場時手順](https://i.imgur.com/EudJpF0.gif)
+
+### 自販機の場合
```
自販機独立記事でもええんちゃう？

graycat27 commented on docs/item/frica.md:
```diff
@@ -0,0 +1,77 @@
+- [FR](/transports/train/fr/)全線全駅
+- 運営設置の自販機
+
+## 購入方法
```
@Nakatai-0322 ここらへん、行間に1行置いてるのは、文ごとに改行したいから？  
md仕様として単純な改行指示は行末に2spaceでいいらしいです

Nakatai-0322 replied:
あー、Markdownの仕様履き違えてましたわ。

graycat27 commented on docs/transports/train/example/cl.md:
```diff
@@ -0,0 +1,50 @@
+
+赤化区間(*1): 全線
+
+高架区間: なし
```
概要で `全線高架化されており` って書いてるのにこれは架空解像度が低い

Nakatai-0322 replied:
なんで赤くなってるんや...wow...

graycat27 reviewed (approved):
私鉄記事書きたいので公営鉄道記事がまずリリースされてほしいです～

Nakatai-0322 commented:
> 私鉄記事書きたいので公営鉄道記事がまずリリースされてほしいです～

担当者からMerge承認取れたのでMergeします。
大変お待たせしました。
